tests/unit/core/reducers/test_addons.js

```diff
@@ -82,4 +82,38 @@ describe('addon reducer', () => {
       },
     });
   });
+
+  it('does not use description from theme_data', () => {
+    // See: https://github.com/mozilla/addons-frontend/issues/2569
+    // TODO: Remove this when
+    // https://github.com/mozilla/addons-frontend/issues/1416 is fixed.
+    const type = ADDON_TYPE_THEME;
+    const state = addons(originalState, {
```

Comment:
I think it will be easy to convert this test file to use real actions instead of simulated actions. I wouldn't suggest that if I thought it would slow you down significantly, I swear.  😁 

There are only 5 tests to fix! Here's [how to use the `loadEntities` action](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/core/test_utils.js#L423).